### PR TITLE
Renaming job to task

### PR DESCRIPTION
**Pull request by pxp9**

### CODE REVIEW

File: src/mod.rs [hunk 1]
```diff
@@ -0,0 +1,13 @@
+pub mod error;
```
Comment (ayrat555) [inline src/mod.rs:1]: I think this file is not required. everything's in lib already?

Comment (pxp9) [reply src/mod.rs]: Compiles without mod.rs file deleting

File: src/queue.rs [hunk 2]
```diff
@@ -808,20 +808,20 @@ mod queue_tests {
     }
 
     #[derive(Serialize, Deserialize)]
-    struct Job {
+    struct PepeTask {
```
Comment (ayrat555) [inline src/queue.rs:811]: Love this name 😄

**Review by ayrat555 — APPROVED** [review]

File: src/queue.rs (hunk 2 above)
Comment (pxp9) [reply src/queue.rs:811]: :D

**Comment by pxp9** [comment]
Can not sign commits , close this pr

**Pull request closed without merging**